### PR TITLE
1st branch neko to develop

### PR DESCRIPTION
**Pull request by ManonBonnan**

### CODE REVIEW

File: firstFile [hunk 1]
```diff
@@ -0,0 +1 @@
+HELLO GIT
```
Comment (MichaelManem) [inline firstFile]: I think this file is useless, why no delete it ?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,27 @@
+node_modules/
```
Comment (MichaelManem) [inline .gitignore]: It's already declared at line 15, should be deleted

**Comment by MichaelManem** [comment]
Approved ✔️

**Comment by Zouif** [comment]
👍

**Pull request merged**